ocaml/ocaml-astring/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup ocaml     1.1
+
+name                ocaml-astring
+version             0.8.5
+revision            0
+categories          ocaml devel
+maintainers         nomaintainer
+license             ISC
+
+description         Alternative String module for OCaml
+long_description    Astring exposes an alternative String module for OCaml. \
+                    This module tries to balance minimality and expressiveness \
+                    for basic, index-free, string processing and provides types \
+                    and functions for substrings, string sets and string maps. \
+                    \
+                    Remaining compatible with the OCaml String module is a \
+                    non-goal. The String module exposed by Astring has \
+                    exception safe functions, removes deprecated and rarely \
+                    used functions, alters some signatures and names, adds a \
+                    few missing functions and fully exploits OCaml's newfound \
+                    string immutability.
+
+homepage            https://erratique.ch/software/astring
+master_sites        https://erratique.ch/software/astring/releases/
```

Nitpick, but this could be:

```suggestion
master_sites        ${homepage}/releases/
```